### PR TITLE
Quac/feat/improve run.py

### PR DESCRIPTION
**Pull request by quac88**
## Description of what you changed
I have updated the game.py file to include a Game class that handles the betting logic. The primary method that needs review in that class is ``betting_round``. I have also updated run.py to show how a game of rhodeisland holdem would be played

## Did you add any new dependencies ex (using a new library, python version, etc)?
__future__

## How did you test your changes?
I tested my changes to make sure the Game class works as expected by running the run.py file, however further unit tests are needed to ensure that no edge cases break the game.

## What issue (if any) does this fix?
This is a feature, not a bug fix.

## Checklist:
- [x] I have read the [contributing.md](contributing.md) rules
- [x] I am follwoing the [branch naming rules](contributing.md#format-for-branch-names)
- [x] I am using symantic release and angular style commits (see [commits.md](docs/commits.md))
- [x] I have tested my changes
- [x] I have added any new dependencies to the [README.md](README.md)
- [x] I have added any new dependencies to the [requirements.txt](requirements.txt)

### CODE REVIEW

File: src/poker/game.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+                if player is None or not player.status: # Skip empty seats and folded players
+                    continue
+
+                if player in players_acted: 
+                    if player != last_raiser or not raise_occurred:
+                        continue
+
+                if current_bet == 0:
+                    available_actions = [1, 3]  # check or raise
```
Comment (quac88) [inline src/poker/game.py]: We should be using enum. Enum will allow us to use strings for better readability, while still being close to the speed of ints. https://docs.python.org/3/library/enum.html

Comment (MichaelAGolden) [reply src/poker/game.py]: I wrote an enum class for actions in my last git branch I stashed. I can submit as a PR if you like

Comment (quac88) [reply src/poker/game.py]: Awesome, a PR for that would be great
